docs/dev/machine-output.adoc

```diff
@@ -1,10 +1,173 @@
-# Machine Readable Output
+= Machine readable output
 
 This document outlines all the machine readable output options and examples.
 
-`odo app describe -o json | jq`
- 
-```json
+See the below table for a list of all possible machine readable output commands:
```

Comment:
@cdrage, iiuc, only those which have machine readable output implemented have the links? Do you think it makes sense to have the links to all of those which have sections outlined below?

Reply:
Yup. Only the ones implemented have the links.